components/ProcessCard.js

```diff
@@ -2,7 +2,7 @@ import 'react'
 
 export default function ProcessCard ({ title, children, icon }) {
   return (
-    <div className='ma3 ma3-m w-25-m w-25-l'>
+    <div className='ma3 ma3-m w-30-m w-30-l h-100  card' >
```

Comment:
what does the card class do?

Reply:
i was trying to use card class of bootstrap
but its not working